ground-server/src/lib/grpcClient.ts

```diff
@@ -0,0 +1,35 @@
+"use server";
+
+import {
+  CommanderClient,
+  type Command,
+  type CommandReply,
+} from "@/proto-out/command";
+import * as grpc from "@grpc/grpc-js";
+import { auth } from "@/app/auth";
+
+const client = new CommanderClient(
+  "localhost:50051",
```

Comment:
You'll want to change this to fill-station:50051 for dev and fill:50051 for prod

Reply:
Changed, but shouldn't localhost:50051 work when running docker compose locally and then just running `npm run dev`?

Reply:
Only if fill station has the correct port exposed